packages/protocol/test-sol/unit/governance/voting/LockedGold.t.sol

```diff
@@ -19,10 +18,9 @@ import "@celo-contracts/governance/test/MockValidators.sol";
 
 import { TestBlocker } from "@test-sol/unit/common/Blockable.t.sol";
 
-contract LockedGoldTest is Test, TestConstants {
+contract LockedGoldTest is Utils {
```

Comment:
even if `Utils` inherits Test, I'd still inherit test here or rename utils as something like `TestWithUtils`, otherwise not instant to know it's a test.